topo_processor/util/conversions.py

```diff
@@ -54,3 +57,17 @@ def quarterdate_to_datetime(value):
         return rfc3339_value
 
     return value
+
+
+def nzt_datetime_to_utc_datetime(date: str) -> datetime:
+    utc_tz = tz.gettz("UTC")
+    nz_tz = tz.gettz("Pacific/Auckland")
+
+    try:
+        nz_time = parser.parse(date).replace(tzinfo=nz_tz)
+    except parser.ParserError as err:
+        raise Exception(f"Not a valid date: {err}") from err
+
+    utc_time = nz_time.astimezone(utc_tz)
+
+    return utc_time
```

Comment:
I think this should return the date in the format `2018-09-30T12:00:00.000Z` (with the military timezone for UTC) rather than the offset in hours.